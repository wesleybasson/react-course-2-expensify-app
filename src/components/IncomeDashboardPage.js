import React from 'react';
import IncomeList from './IncomeList';
import IncomeListFilters from './ExpenseListFilters';
import IncomesSummary from './IncomesSummary';

const IncomeDashboardPage = () => (
    <div>
        <IncomesSummary />
        <IncomeListFilters />
        <IncomeList />
    </div>
);

export default IncomeDashboardPage;