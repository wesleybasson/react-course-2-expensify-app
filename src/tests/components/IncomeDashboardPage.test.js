import React from 'react';
import { shallow } from 'enzyme';
import IncomeDashboardPage from '../../components/IncomeDashboardPage';

test('should render IncomeDashboardPage correctly', () => {
    const wrapper = shallow(<IncomeDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});