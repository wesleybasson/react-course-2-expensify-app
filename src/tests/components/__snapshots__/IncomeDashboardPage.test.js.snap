// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render IncomeDashboardPage correctly 1`] = `
<div>
  <Connect(IncomesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(IncomeList) />
</div>
`;
